.obs/chartfile/operator/templates/channel.yaml

```diff
@@ -1,11 +1,24 @@
-{{ if and .Values.channel .Values.channel.image .Values.channel.tag }}
+{{ if and .Values.channel .Values.channel.image .Values.channel.tag .Values.channel.name }}
 apiVersion: elemental.cattle.io/v1beta1
 kind: ManagedOSVersionChannel
 metadata:
-  name: elemental-channel
+  name: {{ .Values.channel.name }}
   namespace: fleet-default
 spec:
   options:
     image: {{ .Values.channel.image }}:{{ .Values.channel.tag }}
   type: custom
 {{ end }}
+{{ if hasPrefix "registry.opensuse.org" .Values.registryUrl }}
```

Comment:
I'd make this more strict by also including an AND condition like `contains "isv/rancher/elemental"`. This way the check would be valid for branched charts in OBS too.

An example of an unstable channel I created is [here](https://build.opensuse.org/package/show/home:dcassany:branches:isv:Rancher:Elemental:Staging/elemental-unstable-channel-image)

Reply:
That's smart, would anyway move the unstable channel to a separate yaml file (channel-unstable.yaml? channel-dev.yaml? you name it): that would make it easier to automatically drop it from the version we publish in the marketplace

Reply:
Makes sense :+1: